content/blog/2021-09-21-collaborative-experiments.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Running Collaborative Experiments
+date: 2021-09-21
+description: |
+  Sharing experiments with teammates can help you build models more efficiently.
+descriptionLong: |
+  You can use DVC remotes to share experiments and their data across machines.
+picture: 2021-09-21/collaborative-experiments.png
+pictureComment: Running Collaborative Experiments
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/tuning-hyperparameters-with-reproducible-experiments/821
+tags:
+  - MLOps
+  - DVC
+  - Git
+  - Experiments
+  - Collaboration
+---
+
+## Intro
+
+Sharing experiments for machine learning is something that commonly happens. You
+might have a new engineer join the team and you want to get their workspace
+ready quickly. There might be times when someone might have an idea they could
+try on your model if they could access your current progress.
+
+Setting up DVC remotes lets you share all of the data, code, and hyperparameters
+associated with each experiment so anyone can pick up where you left off in the
+training process. We'll go through an example of sharing an experiment with DVC
+remotes.
+
+## Getting the project
+
+To follow along, you can fork
+[this repo](https://github.com/iterative/dvc-learn-project) as one of your own
```

Comment:
It looks like this project is missing the `.dvc` folder, which is needed to recognize that it's a dvc project. Could you either add it, or else ask users to do `dvc init`?

Reply:
Good catch! I'll ask them to run `dvc init`.